src/database/models/InterviewPrep/UserSheet.ts

```diff
@@ -0,0 +1,78 @@
+import { UserSheetModel } from '@/interfaces';
+import { Model, Schema, model, models } from 'mongoose';
+import { databaseModels } from '@/constant';
+
+const UserQuestionSchema = new Schema(
```

Comment:
Here add Type also like you did in UserSheet like new Schema<UserSheetModel>

use UserSheetQuestionModel.